### ISSUE TITLE
Uncaught Error: Cannot decorate a destroyed marker

### ISSUE THREAD

**Issue by pleonex** [facebook/nuclide#666]
### Issue and Steps to Reproduce
It happened closing a file that was open in the Atom start-up (that is, since it was open when closing Atom last time, it was open). The error is the following:

```
Uncaught Error: Cannot decorate a destroyed marker
/home/benito/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.umd.min.js:50

Error: Cannot decorate a destroyed marker
  at DecorationManager.module.exports.DecorationManager.decorateMarker (/usr/share/atom-beta/resources/app.asar/src/decoration-manager.js:192:15)
  at TextEditor.module.exports.TextEditor.decorateMarker (/usr/share/atom-beta/resources/app.asar/src/text-editor.js:1528:37)
  at applyUpdateToEditor (/home/benito/.atom/packages/nuclide/pkg/nuclide-diagnostics-ui/lib/gutter.js:136:14)
  at e.callback [as _next] (/home/benito/.atom/packages/nuclide/pkg/nuclide-diagnostics-ui/lib/main.js:192:55)
  at e.__tryOrUnsub (/home/benito/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.umd.min.js:50:141)
  at e.next (/home/benito/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.umd.min.js:49:63)
  at e._next (/home/benito/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.umd.min.js:47:401)
  at e.next (/home/benito/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.umd.min.js:47:64)
  at c.b.notifyNext (/home/benito/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.umd.min.js:16:355)
  at b._next (/home/benito/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.umd.min.js:10:323)
  at b.e.next (/home/benito/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.umd.min.js:47:64)
  at d._next (/home/benito/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.umd.min.js:245:163)
  at d.e.next (/home/benito/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.umd.min.js:47:64)
  at b.next (/home/benito/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.umd.min.js:40:408)
  at DiagnosticStore._emitFileMessages (/home/benito/.atom/packages/nuclide/pkg/nuclide-diagnostics-common/lib/DiagnosticStore.js:411:25)
  at DiagnosticStore._invalidateFileMessagesForProvider (/home/benito/.atom/packages/nuclide/pkg/nuclide-diagnostics-common/lib/DiagnosticStore.js:180:14)
  at DiagnosticStore.invalidateMessages (/home/benito/.atom/packages/nuclide/pkg/nuclide-diagnostics-common/lib/DiagnosticStore.js:155:14)
  at e._next (/home/benito/.atom/packages/nuclide/pkg/nuclide-diagnostics-store/lib/main.js:202:18)
  at e.__tryOrUnsub (/home/benito/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.umd.min.js:50:141)
  at e.next (/home/benito/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.umd.min.js:49:63)
  at e._next (/home/benito/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.umd.min.js:47:401)
  at e.next (/home/benito/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.umd.min.js:47:64)
  at Function.module.exports.Emitter.simpleDispatch (/usr/share/atom-beta/resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
  at Emitter.module.exports.Emitter.emit (/usr/share/atom-beta/resources/app.asar/node_modules/event-kit/lib/emitter.js:125:28)
  at DiagnosticsProviderBase.publishMessageInvalidation (/home/benito/.atom/packages/nuclide/pkg/nuclide-diagnostics-provider-base/lib/DiagnosticsProviderBase.js:122:21)
  at LinterAdapter._invalidateBuffer (/home/benito/.atom/packages/nuclide/pkg/nuclide-diagnostics-store/lib/LinterAdapter.js:246:29)
  at /home/benito/.atom/packages/nuclide/pkg/nuclide-diagnostics-store/lib/LinterAdapter.js:173:26
  at Function.module.exports.Emitter.simpleDispatch (/usr/share/atom-beta/resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
  at Emitter.module.exports.Emitter.emit (/usr/share/atom-beta/resources/app.asar/node_modules/event-kit/lib/emitter.js:125:28)
  at TextBuffer.module.exports.TextBuffer.destroy (/usr/share/atom-beta/resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:1260:29)
  at TextBuffer.module.exports.TextBuffer.release (/usr/share/atom-beta/resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:1284:14)
  at TextEditor.module.exports.TextEditor.destroyed (/usr/share/atom-beta/resources/app.asar/src/text-editor.js:389:19)
  at TextEditor.module.exports.Model.destroy (/usr/share/atom-beta/resources/app.asar/src/model.js:31:58)
  at Pane.module.exports.Pane.destroyItem (/usr/share/atom-beta/resources/app.asar/src/pane.js:641:18)
  at atom-tabs.TabBarView.onClick (/usr/share/atom-beta/resources/app.asar/node_modules/tabs/lib/tab-bar-view.js:692:17)
```

### Versions
* Atom: 1.10.0-beta4
* Nuclide: 0.161.0
* Client OS: Fedora 23

### Additional Details
* Installed packages (`apm ls --installed`):
```
Dev Packages (1) /home/benito/.atom/dev/packages
└── autocomplete-xml@0.8.3

Community Packages (102) /home/benito/.atom/packages
├── activate-power-mode@0.7.3
├── advanced-open-file@0.16.3
├── aesthetic-ui@2.0.0
├── atom-cscope@0.14.1
├── atom-gmail-checker@0.12.2
├── atom-material-syntax-dark@0.2.5
├── atom-material-ui@1.3.3
├── atom-monokai@0.10.4
├── atom-yeoman@0.3.15
├── autocomplete-clang@0.9.4
├── autocomplete-java@1.2.5
├── autocomplete-python@1.8.6
├── autocomplete-xml@0.8.3
├── blame@0.10.0
├── browser-plus@0.0.75
├── build@0.65.0
├── build-make@0.10.0
├── busy@0.6.0
├── dds-chat@0.0.0
├── docblockr@0.8.0
├── editorconfig@1.4.1
├── file-icons@1.7.18
├── firewatch-syntax@1.0.1
├── fold-comments@0.6.0
├── font-viewer@0.3.1
├── fonts@1.6.0
├── genesis-syntax@1.0.7
├── genesis-ui@0.5.0
├── ginger@0.1.1
├── git-blame@0.4.11
├── git-control@0.8.2
├── git-diff-details@1.3.0
├── git-history@3.2.0
├── git-log@0.4.1
├── git-plus@5.16.2
├── git-projects@1.17.0
├── git-time-machine@1.5.3
├── gruvbox-plus-syntax@1.2.5
├── haskell-grammar@0.4.0
├── highlight-selected@0.11.2
├── hyperclick@0.0.37
├── java-importer@1.4.1
├── json-schema@0.1.15
├── language-ada@1.1.0
├── language-babel@2.32.2
├── language-ini@1.16.0
├── language-latex@0.6.1
├── language-ocaml@1.1.2
├── language-thrift@1.0.2
├── latex@0.34.1
├── latex-plus@0.8.7
├── linter@1.11.16
├── linter-alex@3.2.0
├── linter-chktex@1.3.0
├── linter-clang@3.4.4
├── linter-javac@1.9.4
├── linter-pep8@1.3.0
├── linter-pylama@0.5.6
├── linter-pylint@1.2.1
├── linter-xmllint@1.3.5
├── maximize-panes@0.2.0
├── merge-conflicts@1.4.4
├── minimap@4.24.7
├── minimap-bookmarks@0.3.1
├── minimap-cursorline@0.2.0
├── minimap-find-and-replace@4.5.1
├── minimap-git-diff@4.3.1
├── minimap-highlight-selected@4.4.0
├── minimap-linter@1.1.1
├── minimap-selection@4.4.0
├── monokai-slate@0.5.0
├── move-panes@0.2.0
├── nav-panel@0.0.11
├── navigate@0.1.16
├── nuclide@0.161.0
├── omnisharp-atom@0.30.4
├── package-cop@0.2.10
├── package-sync@1.1.0
├── pdf-view@0.50.0
├── pinned-tabs@0.3.1
├── python-debugger@0.1.0
├── python-indent@0.4.3
├── quick-scroll@2.1.1
├── rainbow-tabs@2.2.1
├── redmond-syntax@1.0.2
├── scroll-percentage@1.2.1
├── seti-syntax@1.0.1
├── seti-ui@1.3.2
├── sort-lines@0.14.0
├── split-diff@1.0.3
├── symbol-gen@1.3.1
├── symbols-tree-view@0.13.2
├── term3@0.22.1
├── tool-bar@1.0.1
├── tool-bar-almighty@0.6.1
├── typing-detector@0.0.0
├── vim-mode@0.65.1
└── vs-blue-ui@0.1.2

└── (empty)
```

**Comment by pleonex** [comment]
Oops, just found that it could be the same as #660

**Comment by matthewwithanm** [comment]
Thanks for the report @pleonex! @nmote does this help you diagnose #660?

**Comment by crucialfelix** [comment]
[Enter steps to reproduce below:]

1. close tab
2. frown at error

Posting the debug info because it now includes the command history.

**Atom Version**: 1.10.2
**Electron Version**: 0.37.8
**System**: Mac OS X 10.11.2
**Thrown From**: [nuclide](https://github.com/facebook/nuclide) package, v0.171.0


### Stack Trace

Uncaught Error: Cannot decorate a destroyed marker

```
At /Users/crucial/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.min.js:50

Error: Cannot decorate a destroyed marker
  at DecorationManager.module.exports.DecorationManager.decorateMarker (/Applications/Atom.app/Contents/Resources/app.asar/src/decoration-manager.js:192:15)
  at TextEditor.module.exports.TextEditor.decorateMarker (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:1540:37)
  at applyUpdateToEditor (/Users/crucial/.atom/packages/nuclide/pkg/nuclide-diagnostics-ui/lib/gutter.js:136:14)
  at d.callback [as _next] (/Users/crucial/.atom/packages/nuclide/pkg/nuclide-diagnostics-ui/lib/main.js:191:55)
  at d.__tryOrUnsub (/Users/crucial/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.min.js:50:395)
  at d.next (/Users/crucial/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.min.js:49:306)
  at d._next (/Users/crucial/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.min.js:48:128)
  at d.next (/Users/crucial/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.min.js:47:323)
  at c.b.notifyNext (/Users/crucial/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.min.js:16:202)
  at b._next (/Users/crucial/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.min.js:10:279)
  at b.d.next (/Users/crucial/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.min.js:47:323)
  at e._next (/Users/crucial/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.min.js:249:79)
  at e.d.next (/Users/crucial/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.min.js:47:323)
  at b.next (/Users/crucial/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.min.js:41:205)
  at DiagnosticStore._emitFileMessages (/Users/crucial/.atom/packages/nuclide/pkg/nuclide-diagnostics-common/lib/DiagnosticStore.js:414:25)
  at DiagnosticStore._invalidateFileMessagesForProvider (/Users/crucial/.atom/packages/nuclide/pkg/nuclide-diagnostics-common/lib/DiagnosticStore.js:180:14)
  at DiagnosticStore.invalidateMessages (/Users/crucial/.atom/packages/nuclide/pkg/nuclide-diagnostics-common/lib/DiagnosticStore.js:155:14)
  at d._next (/Users/crucial/.atom/packages/nuclide/pkg/nuclide-diagnostics-store/lib/main.js:202:18)
  at d.__tryOrUnsub (/Users/crucial/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.min.js:50:395)
  at d.next (/Users/crucial/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.min.js:49:306)
  at d._next (/Users/crucial/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.min.js:48:128)
  at d.next (/Users/crucial/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.min.js:47:323)
  at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
  at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:125:28)
  at DiagnosticsProviderBase.publishMessageInvalidation (/Users/crucial/.atom/packages/nuclide/pkg/nuclide-diagnostics-provider-base/lib/DiagnosticsProviderBase.js:122:21)
  at LinterAdapter._invalidateBuffer (/Users/crucial/.atom/packages/nuclide/pkg/nuclide-diagnostics-store/lib/LinterAdapter.js:246:29)
  at /Users/crucial/.atom/packages/nuclide/pkg/nuclide-diagnostics-store/lib/LinterAdapter.js:173:26
  at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/node_modules/event-kit/lib/emitter.js:25:14)
  at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/node_modules/event-kit/lib/emitter.js:129:28)
  at TextBuffer.module.exports.TextBuffer.destroy (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:1288:29)
  at TextBuffer.module.exports.TextBuffer.release (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:1312:14)
  at TextEditor.module.exports.TextEditor.destroyed (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:401:19)
  at TextEditor.module.exports.Model.destroy (/Applications/Atom.app/Contents/Resources/app.asar/src/model.js:31:58)
  at Pane.module.exports.Pane.destroyItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:641:18)
  at Pane.module.exports.Pane.destroyActiveItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:619:12)
  at Workspace.module.exports.Workspace.destroyActivePaneItem (/Applications/Atom.app/Contents/Resources/app.asar/src/workspace.js:703:35)
  at Workspace.module.exports.Workspace.closeActivePaneItemOrEmptyPaneOrWindow (/Applications/Atom.app/Contents/Resources/app.asar/src/workspace.js:742:21)
  at atom-workspace.commandRegistry.add.core:close (/Applications/Atom.app/Contents/Resources/app.asar/src/register-default-commands.js:217:32)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:580:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:388:22)
  at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:98:36)
  at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:3:61)

```

### Commands

```
     -0:55.3.0 core:confirm (atom-text-editor.omnisearch-pane.mini.is-focused)
  4x -0:55.3.0 blur (atom-text-editor.omnisearch-pane.mini.is-focused)
     -0:55.1.0 tabs:keep-preview-tab (atom-text-editor.editor.vim-mode.normal-mode)
  2x -0:46.4.0 vim-mode:move-left (atom-text-editor.editor.vim-mode.normal-mode.enable-nuclide-find-references.is-focused)
     -0:45.7.0 vim-mode:move-right (atom-text-editor.editor.vim-mode.normal-mode.enable-nuclide-find-references.is-focused)
     -0:44.9.0 vim-mode:activate-insert-mode (atom-text-editor.editor.vim-mode.normal-mode.enable-nuclide-find-references.is-focused)
  4x -0:44.3.0 editor:select-to-end-of-word (atom-text-editor.editor.vim-mode.enable-nuclide-find-references.is-focused.insert-mode)
     -0:43 core:select-right (atom-text-editor.editor.vim-mode.enable-nuclide-find-references.is-focused.insert-mode)
     -0:42.5.0 core:backspace (atom-text-editor.editor.vim-mode.enable-nuclide-find-references.is-focused.insert-mode)
  3x -0:35.9.0 core:move-down (atom-text-editor.editor.vim-mode.enable-nuclide-find-references.is-focused.insert-mode)
     -0:31.8.0 core:save (atom-text-editor.editor.vim-mode.enable-nuclide-find-references.is-focused.insert-mode)
     -0:15.2.0 find-and-replace:use-selection-as-find-pattern (atom-text-editor.editor.vim-mode.enable-nuclide-find-references.is-focused.insert-mode)
     -0:15 find-and-replace:find-next (atom-text-editor.editor.vim-mode.enable-nuclide-find-references.is-focused.insert-mode)
  3x -0:14.9.0 blur (atom-text-editor.editor.vim-mode.enable-nuclide-find-references.is-focused.insert-mode)
     -0:00.3.0 core:close (atom-text-editor.editor.vim-mode.enable-nuclide-find-references.is-focused.insert-mode)
  4x -0:00.3.0 blur (atom-text-editor.editor.vim-mode.enable-nuclide-find-references.is-focused.insert-mode)
```

### Config

```json
{
  "core": {
    "customFileTypes": {
      "source.ini": [
        ".buckconfig",
        ".flowconfig",
        ".hgrc"
      ],
      "source.json": [
        "BUCK.autodeps"
      ],
      "source.python": [
        "BUCK"
      ]
    },
    "disabledPackages": [
      "cute-cursor",
      "language-go",
      "language-java",
      "term",
      "language-supercollider",
      "linter-rst",
      "rst-preview",
      "rst-preview",
      "ide-haskell",
      "remote-edit",
      "atomatigit",
      "minimap-highlight-selected",
      "atomic-emacs",
      "autoflow",
      "autosave",
      "background-tips",
      "easy-motion",
      "gist-it",
      "git-blame",
      "git-difftool",
      "term2",
      "web-view",
      "css-color-highlighting",
      "selection-count",
      "fancy-new-file",
      "minimap-find-and-replace",
      "mobile-preview",
      "my-package",
      "org",
      "paredit",
      "atom-color-highlight",
      "autocomplete-jedi",
      "coffee-lint",
      "python-jedi",
      "next-occurrence",
      "linter-scss-lint",
      "linter-coffeelint",
      "grunt-runner",
      "typewriter-sounds",
      "ide-flow",
      "autocomplete-paths",
      "auto-copyright",
      "dash",
      "node-debugger",
      "processing",
      "language-ruby-on-rails",
      "glist",
      "language-docker",
      "typewriter",
      "language-javascript-jsx",
      "linter-jshint",
      "atom-ternjs",
      "language-gfm",
      "atom-lupa",
      "tabs-to-spaces",
      "symbols-tree-view",
      "ctags-status",
      "tidalcycles",
      "linter-python",
      "imdone-atom",
      "language-python",
      "open-recent",
      "symbol-gen",
      "language-clojure",
      "atom-terminal",
      "tree-view",
      "linter"
    ],
    "followSymlinks": true,
    "ignoredNames": [
      "*.pyc",
      ".DS_Store",
      "tags"
    ],
    "packagesWithKeymapsDisabled": [
      "emmet",
      "schelp-syntax"
    ],
    "themes": [
      "one-light-ui",
      "one-light-syntax"
    ]
  },
  "nuclide": {
    "nuclide-arcanist": {},
    "nuclide-buck-files": {},
    "nuclide-flow": {
      "enableTypeHints": true
    },
    "nuclide-format-js": {},
    "nuclide-health": {},
    "nuclide-home": {
      "showHome": false
    },
    "nuclide-ocaml": {},
    "use": {
      "nuclide-arcanist": false,
      "nuclide-blame": false,
      "nuclide-blame-ui": false,
      "nuclide-buck-files": false,
      "nuclide-clang-atom": false,
      "nuclide-language-hack": false,
      "nuclide-objc": false,
      "nuclide-ocaml": false
    }
  }
}
```

### Installed Packages

```coffee
# User
advanced-open-file, v0.16.4 (inactive)
afterglow-monokai-syntax, v1.3.4 (inactive)
archiver, v0.0.0 (inactive)
atom-beautify, v0.29.13 (active)
atom-dark-fusion-syntax, v1.2.4 (inactive)
atom-django, v0.3.2 (active)
atom-easy-jsdoc, v4.7.1 (active)
atom-fuzzy-grep, v0.14.0 (inactive)
atom-material-syntax, v0.4.6 (inactive)
atom-material-syntax-light, v0.4.5 (inactive)
atom-material-ui, v1.3.6 (inactive)
atom-terminal, v0.8.0 (inactive)
atom-ternjs, v0.15.0 (inactive)
auto-update-packages, v1.0.1 (active)
autocomplete-modules, v1.6.3 (active)
autocomplete-paths, v1.0.2 (inactive)
autoupdate-packages, v1.3.1 (active)
browser-plus, v0.0.76 (active)
busy, v0.7.0 (active)
clipboard-plus, v0.5.1 (active)
color-picker, v2.2.2 (active)
copy-as-rtf, v0.9.3 (inactive)
cursor-history, v0.6.1 (active)
dash, v1.6.2 (inactive)
django-templates, v0.5.0 (active)
docblockr, v0.8.5 (active)
editorconfig, v1.4.1 (active)
emmet, v2.4.3 (active)
envy-theme, v0.1.1 (inactive)
es6-javascript, v1.0.0 (inactive)
find-and-replace-under-cursor, v0.3.0 (inactive)
find-selection, v0.16.0 (inactive)
fizzy, v0.16.0 (inactive)
focus-dark, v1.1.0 (inactive)
focus-light, v1.1.0 (inactive)
fold-navigator, v0.0.618 (inactive)
freshcut-theme, v0.3.0 (inactive)
git-plus, v5.17.1 (inactive)
git-time-machine, v1.5.3 (inactive)
graphite-ui, v0.15.1 (inactive)
highlight-selected, v0.11.2 (active)
hip-light-ui-theme, v1.1.1 (inactive)
html-entities, v0.4.0 (active)
incremental-search, v5.1.0 (inactive)
isotope-ui, v2.7.1 (inactive)
language-clojure, v0.21.0 (inactive)
language-cmake, v0.2.0 (active)
language-ini, v1.16.0 (active)
language-jade, v0.7.2 (active)
language-markdown, v0.16.3 (active)
language-nginx, v0.6.2 (active)
language-restructuredtext, v1.0.0 (active)
lines, v0.13.1 (inactive)
linter, v1.11.16 (inactive)
linter-csslint, v1.3.2 (active)
linter-eslint, v7.3.2 (active)
linter-flake8, v1.13.4 (active)
linter-htmlhint, v1.3.0 (active)
linter-jade, v0.3.2 (active)
linter-json-lint, v0.1.1 (active)
linter-pylama, v0.7.2 (active)
linter-sass-lint, v1.7.1 (active)
MagicPython, v0.5.15 (active)
markdown-folder, v0.5.0 (inactive)
markdown-pdf, v1.5.0 (inactive)
markdown-writer, v2.6.1 (active)
maximize-panes, v0.2.0 (inactive)
move-panes, v0.2.0 (inactive)
narrow, v0.6.0 (active)
navigate, v0.1.16 (active)
nucleus-dark-ui, v0.10.2 (inactive)
nuclide, v0.171.0 (active)
octocat-syntax, v0.1.5 (inactive)
open-in, v4.0.0 (inactive)
open-in-sourcetree, v0.1.8 (inactive)
open-terminal-here, v2.1.4 (inactive)
open-this, v0.2.0 (inactive)
package-cop, v0.2.10 (active)
pain-split, v1.4.0 (active)
pandoc-convert, v1.1.0 (active)
pane-layout-plus, v0.7.3 (inactive)
pdf-view, v0.50.0 (active)
pigments, v0.35.1 (active)
postcss-sorting, v2.7.0 (active)
preview, v0.16.9 (inactive)
processing-language, v0.2.0 (active)
project-manager, v3.2.3 (active)
pubster-syntax, v1.4.1 (inactive)
python-isort, v0.0.7 (active)
python-tools, v0.6.8 (active)
quick-file-actions, v0.1.1 (inactive)
quote-words, v0.1.1 (inactive)
railscast-theme, v0.3.0 (inactive)
recent-files-fuzzy-finder, v0.2.4 (active)
regex-railroad-diagram, v0.16.0 (active)
release-notes, v0.53.0 (active)
schelp-syntax, v0.6.3 (inactive)
set-syntax, v0.3.0 (active)
seti-monokai, v0.3.1 (inactive)
seti-syntax, v1.0.1 (inactive)
seti-ui, v1.3.2 (inactive)
show-in-system, v0.4.0 (inactive)
sort-lines, v0.14.0 (inactive)
string-case, v0.1.0 (inactive)
supercollider, v0.7.2 (inactive)
tab-control, v0.6.9 (active)
tab-switcher, v1.5.3 (active)
toggle-quotes, v1.0.0 (inactive)
tomorrow, v1.0.0 (inactive)
tool-bar, v1.0.1 (active)
unity-ui, v2.1.5 (inactive)
vim-mode, v0.65.1 (active)
Zen, v0.16.4 (inactive)
zenburn-theme, v1.3.0 (inactive)
zentabs, v0.8.7 (active)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.52.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.44.0 (inactive)
base16-tomorrow-dark-theme, v1.1.0 (inactive)
base16-tomorrow-light-theme, v1.1.1 (inactive)
one-dark-ui, v1.5.0 (inactive)
one-light-ui, v1.5.0 (active)
one-dark-syntax, v1.3.0 (inactive)
one-light-syntax, v1.3.0 (active)
solarized-dark-syntax, v1.0.2 (inactive)
solarized-light-syntax, v1.0.2 (inactive)
about, v1.5.3 (active)
archive-view, v0.61.1 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.11.2 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.31.1 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (inactive)
background-tips, v0.26.1 (inactive)
bookmarks, v0.41.0 (active)
bracket-matcher, v0.82.1 (active)
command-palette, v0.38.0 (active)
deprecation-cop, v0.54.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.39.0 (active)
find-and-replace, v0.201.0 (active)
fuzzy-finder, v1.3.0 (active)
git-diff, v1.1.0 (active)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.1 (active)
image-view, v0.58.2 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.1 (inactive)
markdown-preview, v0.158.0 (active)
metrics, v0.53.1 (active)
notifications, v0.65.0 (active)
open-on-github, v1.2.0 (inactive)
package-generator, v1.0.0 (inactive)
settings-view, v0.242.2 (active)
snippets, v1.0.2 (active)
spell-check, v0.67.1 (active)
status-bar, v1.4.1 (active)
styleguide, v0.47.0 (active)
symbols-view, v0.113.0 (inactive)
tabs, v0.100.2 (active)
timecop, v0.33.2 (active)
tree-view, v0.208.2 (inactive)
update-package-dependencies, v0.10.0 (active)
welcome, v0.34.0 (active)
whitespace, v0.33.0 (active)
wrap-guide, v0.38.1 (active)
language-c, v0.52.1 (active)
language-clojure, v0.21.0 (inactive)
language-coffee-script, v0.47.2 (active)
language-csharp, v0.12.1 (active)
language-css, v0.37.1 (active)
language-gfm, v0.88.0 (inactive)
language-git, v0.15.0 (active)
language-go, v0.42.1 (inactive)
language-html, v0.45.1 (active)
language-hyperlink, v0.16.0 (active)
language-java, v0.23.0 (inactive)
language-javascript, v0.119.0 (active)
language-json, v0.18.2 (active)
language-less, v0.29.5 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.35.0 (active)
language-php, v0.37.2 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.0 (inactive)
language-ruby, v0.69.0 (active)
language-ruby-on-rails, v0.25.0 (inactive)
language-sass, v0.56.0 (active)
language-shellscript, v0.22.4 (active)
language-source, v0.9.0 (active)
language-sql, v0.23.0 (active)
language-text, v0.7.1 (active)
language-todo, v0.28.0 (active)
language-toml, v0.18.0 (active)
language-xml, v0.34.9 (active)
language-yaml, v0.26.0 (active)

# Dev
No dev packages
```

**Comment by hansonw** [comment]
Should be fixed by acc805f (hopefully will be released next Monday).

**Comment by hansonw** [comment]
175 is out, please re-open if it happens again.
